.travis.yml

```diff
@@ -1,32 +1,37 @@
 language: java
-install: true
+
 jdk:
-- openjdk8
+  - openjdk8
+
 before_cache:
-- rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
-- rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+
 cache:
   directories:
-  - "$HOME/.gradle/caches/"
-  - "$HOME/.gradle/wrapper/"
+    - "$HOME/.gradle/caches/"
+    - "$HOME/.gradle/wrapper/"
+
 stages:
-- name: test
-- name: build
-- name: release
-  if: branch = release
+  - name: build
+  - name: release
+    if: branch = release
+
 jobs:
   include:
-  - stage: test
-    script: "./gradlew test"
-  - stage: build
-    script: "./gradlew build -x test"
-  - stage: release
-    script: "./gradlew publish -x test -Ddisablesnapshot=true -Dnexus.user=$NEXUS_USER -Dnexus.key=$NEXUS_KEY
-      -Dbuild.number=$TRAVIS_BUILD_NUMBER"
+    - stage: build
+      before_install:
+        git checkout ${TRAVIS_BRANCH} # to fix travis checking out HEAD branch (breaks grgit)
+      install: ./gradlew assemble
+      script: ./gradlew check
+    - stage: release
+      script: ./gradlew publish -Ddisablesnapshot=true -Dnexus.user=$NEXUS_USER -Dnexus.key=$NEXUS_KEY
+        -Dbuild.number=$TRAVIS_BUILD_NUMBER
+
 deploy:
   provider: releases
   api_key:
-    secure: A8JANJAzvBevIZD9F/J23iiRhBsRUKwi1YdKhstTP9WwsMUl6aM3NFwBQCV4bWSpRwNfvYAzUsRJVnqt2PyoSoVTnFIeXcCNQi6tiQBuakSfvNRueKlc4dLwENxBHhz1WvUui0oJocmAGDwmbjcPhtl/z5MDZpBZbL+X0JTym1Orza/BvaQhBFXI7w8lQG3okgoBXtB4B/yJV19m/X019fqDUnn1Vttxw+8ezVgwJmHTxyr/Jr65H9pwW3XtwfOJ2XvnJFKALpQDNZj6D2uLMZOT9Hljc6fvrQIYYbNJ4foJ46MlVBUDt7aq2Y3FwnM4hAwT1feJYrIOKq2mR0mzDGDsBwdTSDqnnCI06vpkqG0iBF5JtARU76BW5V40iZ98A2NvqH0Sg7qCtgU19n8VFjaTvD0g/bhe4PIsRkeFNKA5YySj1khAkUOXjfxn+BTtAAxQQDSp3rLw/k7o2pYyLzPMNiyKUgaQv6pRGBnJi5if6ujimKmC9c2UuYQF9S5HcOi5an+/vTY4/s1RmxKLthfBE5QJbTGggOEUYqWDFPSvReo7976FI3kodzMN0Fzu+Ib9zyzrKEEgXD5Dg4oFYA+7oWnaAGVLq7fiejssA96oZa7Hb6ODfmHl6XzoV6AmOT2/SlXstgj4CczJgp5D8cyLcRQP5DaO0UhbEY2Euvc=
+    secure: hi62cv1hYPi6S8ii1PI3C9EpTnrgB/06rO+uFsPGJQeQvJIkP5fJ07wDhih+wqlA81T4QE6xtqTddn/i6QquorzGDHv9MoO7l42kedyXgAEMqcuw90do6cRMGChpz66VqQvFUaE+6qzn46HwasB2SxN0U8OmUUKBiz142o51zNPI0D+oOeErEGhqmsKq71npK/YlLIWyFEHaoxTq6vn+zynFatJ0d4eD/fbrVDvl+8CGVNuXscGrSqvszkAUl0yFOXli7zh3aCkQYm6nXzJUNL7Q5GTfcfCC/gkY5ugYBfst16d7mGydyxrI6i5+NhuwDgPSJ3x4MuGgMrc6fX00AEEPnNsvkRmb2P8NVZNruWihUF4ir/pov1GgElgIPFsiS+NCT/EbLTnmlxnqDrJM4dYET5s7QkR07V1sNAjBnRhXgzu0ZKvXtN+n7RktriOmV4BW3nHGHlbp2P50c2h4ItZcbrJRd79ZuTSLBKpgcYiulgG5zN8qWYTsnmA/XIBI+hH2wM2M5PRjiyyvifoDJSgPRAngjj2o6YMct+J/jpAB8NhlaM4VUZT2dtkfk1UMO5AvRxx5xeVs5vnrNatg7FyswcGQQZE/YYBOaP3JRqrvVHCbWz/kLSfxXL9OLmQ8KyYkZzXgG8t/wBPnhtzoNLOanw6anE6qaWTM1EhX4MM=
```

Comment:
Why did this change?

Reply:
I made some changes to the Craco travis configuration and then just copied that into the other project's files. So the API key is specific to the project? I guess I can just delete the deployment part for now since we don't deploy on Nexus anymore.